src/functions/env.ts

```diff
@@ -196,11 +196,15 @@ function findEnvfiles(functionsSource: string, projectId: string, projectAlias?:
  *
  * @return True if there are any user-specified environment variables
  */
-export function hasUserEnvs(
-  functionsSource: string,
-  projectId: string,
-  projectAlias?: string
-): boolean {
+export function hasUserEnvs({
```

Comment:
I don't generally like inlining types. Does it happen elsewhere in the codebase?

Reply:
Great callout. Named this type instead.